### PR TITLE
JSON

### PR DESCRIPTION
**Pull request by AbdealiJK**

### CODE REVIEW

File: coalib/coala_json.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            for i in [1, 2]:
+                for key, value in section_result[i].items():
+                    for result in value:
+                        results_for_section.append({
+                            "origin": result.origin,
+                            "message": result.message,
+                            "file": result.file,
+                            "line_number": result.line_nr,
+                            "severity": result.severity,
+                            "debug_msg": result.debug_msg})
```
Comment (sils1297) [inline coalib/coala_json.py]: didn't we want to move this conversion into the result class?

Comment (AbdealiJK) [inline coalib/coala_json.py]: Commit isn't complete yet :)

**Comment by AbdealiJK** [comment]
SOme issue with appveyor - it's not building anything

**Comment by AbdealiJK** [comment]
The scrutinizer issue cannot be avoided.

**Comment by AbdealiJK** [comment]
@rultor merge

**Pull request merged**